src/index.html

```diff
@@ -1,49 +1,63 @@
-      <p>No wonder we are a mess!</p>
-      <p>
-        What we really need is to get outside...in fact SLEEP outside. Our
-        mission is to provide you with everything you need to do this
-        comfortably at an affordable price.
-      </p>
-    </section>
-    <section class="product-categories">
-      <a href="product_listing/index.html?category=Tents">
-        <img src="https://byui-cit.github.io/wdd330/images/category-icons/noun_Tent_3544767.svg" alt="Tent Icon from Noun Project: Mustofa Bayu">
-      </a>
-      <a href="product_listing/index.html?category=Sleeping-bags">
-        <img src="https://byui-cit.github.io/wdd330/images/category-icons/noun_Sleeping%20Bag_3544775.svg" alt="Sleeping Bag Icon from Noun Project: Mustofa Bayu">
-      </a>
-      <a href="product_listing/index.html?category=Backpacks">
-        <img src="https://byui-cit.github.io/wdd330/images/category-icons/noun_carrier%20bag_3544771.svg" alt="Backpack Icon from Noun Project: Mustofa Bayu">
-      </a>
-      <a href="product_listing/index.html?category=Hammocks">
-        <img src="https://byui-cit.github.io/wdd330/images/category-icons/noun_Hammock_791143.svg" alt="Hammock Icon from Noun Project: Paul Richard">
-      </a>
-    </section>
-  </main>
-  <footer>&copy;NOT a real business</footer>
-</body>
-
-</html>
\ No newline at end of file
+      <section class="mission">
+        <p>We spend too much time these days indoors...sitting usually.</p>
+        <p>No wonder we are a mess!</p>
+        <p>
+          What we really need is to get outside...in fact SLEEP outside. Our
+          mission is to provide you with everything you need to do this
+          comfortably at an affordable price.
+        </p>
+      </section>
+      <section class="product-categories">
+        <a href="product_listing/index.html?category=Tents">
+          <img
+            src="https://byui-cit.github.io/wdd330/images/category-icons/noun_Tent_3544767.svg"
+            alt="Tent Icon from Noun Project: Mustofa Bayu"
+          />
+        </a>
+        <a href="product_listing/index.html?category=Sleeping-bags">
+          <img
+            src="https://byui-cit.github.io/wdd330/images/category-icons/noun_Sleeping%20Bag_3544775.svg"
+            alt="Sleeping Bag Icon from Noun Project: Mustofa Bayu"
+          />
+        </a>
+        <a href="product_listing/index.html?category=Backpacks">
+          <img
+            src="https://byui-cit.github.io/wdd330/images/category-icons/noun_carrier%20bag_3544771.svg"
+            alt="Backpack Icon from Noun Project: Mustofa Bayu"
+          />
+        </a>
+        <a href="product_listing/index.html?category=Hammocks">
+          <img
+            src="https://byui-cit.github.io/wdd330/images/category-icons/noun_Hammock_791143.svg"
+            alt="Hammock Icon from Noun Project: Paul Richard"
+          />
+        </a>
+      </section>
+    </main>
+    <footer>&copy;NOT a real business</footer>
```

Comment:
Somehow this got missed in my header/footer PR. Can you swap this for the new footer?
```suggestion
    <footer id="main-footer"></footer>
```